rmgpy/rmg/model.py

```diff
@@ -872,9 +879,6 @@ def processNewReactions(self, newReactions, newSpecies, pdepNetwork=None, genera
                     if rxn in self.edge.reactions:
                         self.edge.reactions.remove(rxn)
 
-            if not numpy.isinf(self.toleranceThermoKeepSpeciesInEdge) and spcs != []: #do thermodynamic filtering
```

Comment:
You can also remove the creation of the `spcs` variable above.

Reply:
done